pkg/consistenthash/maglev/maglev.go

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package maglev
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"unsafe"
+
+	"github.com/cilium/ebpf"
+
+	cluster_v2 "kmesh.net/kmesh/api/v2/cluster"
+	"kmesh.net/kmesh/api/v2/endpoint"
+	"kmesh.net/kmesh/pkg/utils/hash"
+)
+
+const (
+	DefaultTableSize    uint64 = 16381
+	DefaultHashSeed            = "JLfvgnHc2kaSUFaI"
```

Comment:
Is this a common practice to use a DefaultHashSeed?

Reply:
Only if this seed can ensure random and unique,  to generate different hash values， and It can work.